packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -1506,6 +1537,20 @@ export const DialogueMatchingPage = ({classes}: {
 
   const prompt = "Opt-in to LessWrong team viewing your checks, to help proactively suggest and facilitate dialogues" 
 
+  if (!!activeDialogueCheck && !!activeTargetUser) {
```

Comment:
I was confused by what this was supposed to be doing based on the naming of `activeDialogueCheck` and `activeTargetUser` until I figured out what the whole PR was doing.  Would rename